src/commands/stats.js

```diff
@@ -0,0 +1,54 @@
+const { Command } = require('../../lib/Command.js');
+const command = new Command( ['stats'], 0, function ( { msg } ) {
+    // eslint-disable-next-line no-magic-numbers
+    const memoryUsage = (process.memoryUsage().heapUsed / 1024 / 1024).toFixed(2);
+    const cpu = require('os').cpus().map(i => i.model);
+    const cpuLength = cpu.length;
+    const cpuType = cpu[0];
+    const uptime = this.utils.elapsed(this.bot.uptime);
+    this.sendMsg(msg.channel, {
+        embed: {
+            title: `${this.bot.user.username} statistics`,
+            color: 0xFFFFFF,
+            thumbnail: {
+                url: this.bot.user.avatarURL,
+            },
+            description: `[Git](${require('../../package.json').repository.url}) | [Discord](https://discord.gg/${require('../../config.json').discord}/)`,
```

Comment:
require package.json once and use it, instead of requiring many time in your code.
That will at least be better when reading the code, but will also help performance wise.